### DISCUSSION TITLE
Custom Icons for pages

### DISCUSSION THREAD

**Discussion by n8himmel** [GitbookIO/community#648] [Feature Requests]
As a visual person, I miss the ability to decorate pages with custom icons. Of course, there are emojis, but that's very limited and you often have to resort to an icon that does not exactly represent the topic of the page. It is already possible to use custom icons for spaces, and the same feature for pages would be greatly appreciated. This would make it much easier to find the right page at a glance.

![image](https://github.com/GitbookIO/community/assets/26195744/f6422434-9eab-424b-ab07-dfdf651b6840)
_Screenshot: Notion_

**Comment by addisonschultz** [comment]
Hello @n8himmel ! We've just shipped this feature into GitBook, and is now available for you to add! 

Here's the docs: https://docs.gitbook.com/content-editor/editor/content-structure/content-in-a-space#add-an-icon-or-emoji-to-your-page

You can read more about adding icons and other updates we've made here: https://github.com/orgs/GitbookIO/discussions/675

**Comment by denisemcg** [comment]
The new feature mentioned above doesn't support **custom** icons, i.e. my own brand icons that I want to upload and use on my pages.

**Comment by eric-maniloff** [comment]
it looks like all of our favicons next to title at the top of the browser all changed to a square letter "E" (I think for english?) for ALL of our spaces. before the upgrade it was using our custom brand icons for the favicons....any way to get those back?

**Reply by eric-maniloff** [reply]
![image](https://github.com/user-attachments/assets/ab78e905-3908-4e50-93cb-324b7567a27e)

**Reply by addisonschultz** [reply]
Hey @eric-maniloff - can you reach out to us at support (at) gitbook.com? We can help you look into this there!
